js/post.js

```diff
@@ -2,14 +2,23 @@
 	'use strict';
 
 	$( '.rwmb-post-add-button' ).rwmbModal( {
+		$postId: null,
 		removeElement: '#editor .interface-interface-skeleton__footer, .edit-post-fullscreen-mode-close',
-		callback: function( $modal ) {
-			const isBlockEditor = $modal.find( 'body' ).hasClass( 'block-editor-page' );
-
-			if ( !isBlockEditor ) {
+		callback: function ( $modal ) {
+			if ( !this.isBlockEditor ) {
```

Comment:
Biến `this` này mình hiểu là lấy từ đâu ra em nhỉ? Đọc code ko clear lắm, dễ xảy ra lỗi.

Reply:
Em đưa vào options của modal.js. Vì cái này cũng có dùng ở trong modal.
![image](https://user-images.githubusercontent.com/3814519/229725284-c0e77fff-59c0-438b-8ff6-11072a15170d.png)